Gulpfile.js

```diff
@@ -31,16 +36,14 @@ var options = {
 };
 
 gulp.task('lint', function () {
-    return gulp.src(['./assets/sass/**/*.scss', '!./assets/sass/vendor/**/*.scss'])
-        .pipe(scsslint({
-            'config': '.scss-lint.yml',
-            'reporterOutputFormat': 'Checkstyle',
-            'filePipeOutput': 'scssReport.xml'
+    return gulp.src([paths.scssDir, '!./assets/sass/vendor/**/*.scss'])
+        .pipe(sassLint({
+          configFile: '.sass-lint.yml'
         }))
         .pipe(gulp.dest(
```

Comment:
We don't need to do this for this linter, it copies all the sass files. 
We do need to add .pipe(sassLint.format()) else it just tells you which file failed, not what line.